Testing/SystemTests/tests/framework/ISISReflectometryAutoreductionTest.py

```diff
@@ -280,5 +288,5 @@ def SortRuns(tupsort):
 
 
 def CreateTransmissionWorkspaces(run1, run2, scale=False):
-    CreateTransmissionWorkspaceAuto(run1, OutputWorkspace=TransmissionWorkspaceName(run1), StartOverlap=10, EndOverlap=12)
-    CreateTransmissionWorkspaceAuto(run2, OutputWorkspace=TransmissionWorkspaceName(run2), StartOverlap=10, EndOverlap=12)
```

Comment:
Do you know how this was working before?

Reply:
I do not!

Reply:
Ah, it looks like the function that this code is inside is unused

Reply:
Don't worry, we'll just get the imports done in this pr